kernel/usage.c

```diff
@@ -170,6 +170,13 @@ void z_sched_cpu_usage(uint8_t cpu_id, struct k_thread_runtime_stats *stats)
 }
 #endif /* CONFIG_SCHED_THREAD_USAGE_ALL */
 
+void z_sched_thread_reset_long_frame(struct k_thread *thread)
```

Comment:
Shouldn't this be called from somewhere to clear the long-usage statistics after printing?

Reply:
You are correct.  Missed that piece in moving from internal.  Thanks for catching it!